src/libstd/net/ip.rs

```diff
@@ -342,6 +342,34 @@ impl Ipv4Addr {
         }
     }
 
+    /// Creates a new IPv4 address with the address pointing to localhost: 127.0.0.1.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
I believe you need to add `# #![feature(ip)]` to this and the other new examples in order for the doc tests to work. The first `#` hides the line from the docs (which you might not want to do to make it clear that it is needed for this usage).